runbotics/runbotics-orchestrator-ui/src/src-app/views/users/UsersRegisterView/UsersRegisterView.tsx

```diff
@@ -1,9 +1,145 @@
-import React, { FunctionComponent } from 'react';
+import React, { VFC, useState, useEffect } from 'react';
 
-const UsersRegisterView: FunctionComponent = () => {
+import { useSearchParams } from 'next/navigation'; 
+import { useRouter } from 'next/router';
+import { useSnackbar } from 'notistack';
+import { useSelector } from 'react-redux';
+import { Role, IUser } from 'runbotics-common';
+
+import useTranslations from '#src-app/hooks/useTranslations';
+import useUserSearch from '#src-app/hooks/useUserSearch';
+import { useDispatch } from '#src-app/store';
+import { usersActions, usersSelector } from '#src-app/store/slices/Users';
+
+import { DefaultValue, ROWS_PER_PAGE } from '../UsersBrowseView/UsersBrowseView.utils';
+import UsersRegisterTable from './UsersRegisterTable';
+import { StyledButtonsContainer, StyledButton, DeleteButton, StyledActionsContainer, StyledTextField } from './UsersRegisterView.styles';
+
+interface SelectedRoles { [id: number]: Role };
+interface MapActivatedUserParams {
+    id: number;
+    login: string;
+    roles: Role[];
+    activated: boolean;
+}
+
+const UsersRegisterView: VFC = () => {
```

Comment:
VFC stands for void function component, but here you're returning children -> change to FC